utils/lite-components-testkit/src/main/scala/pl/touk/nussknacker/engine/lite/util/test/LiteTestScenarioRunner.scala

```diff
@@ -0,0 +1,69 @@
+package pl.touk.nussknacker.engine.lite.util.test
+
+import com.typesafe.config.Config
+import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.component.ComponentDefinition
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.context.transformation.{NodeDependencyValue, SingleInputGenericNodeTransformation}
+import pl.touk.nussknacker.engine.api.definition.{NodeDependency, TypedNodeDependency}
+import pl.touk.nussknacker.engine.api.process.{SinkFactory, Source, SourceFactory}
+import pl.touk.nussknacker.engine.api.typed.typing.{Typed, TypingResult}
+import pl.touk.nussknacker.engine.graph.EspProcess
+import pl.touk.nussknacker.engine.lite.api.interpreterTypes.{ScenarioInputBatch, SourceId}
+import pl.touk.nussknacker.engine.lite.api.utils.sinks.LazyParamSink
+import pl.touk.nussknacker.engine.lite.api.utils.sources.BaseLiteSource
+import pl.touk.nussknacker.engine.lite.util.test.LiteTestScenarioRunner.{sinkName, sourceName}
+import pl.touk.nussknacker.engine.util.test.{ModelWithTestComponents, TestScenarioRunner}
+
+import scala.reflect.ClassTag
+
+object LiteTestScenarioRunner {
+
+  val sourceName = "test-source"
```

Comment:
Can you add comment to `runWithData` that developer should create scenario with:
```
      .source("source", LiteTestScenarioRunner.sourceName)
(...)
      .emptySink("sink", LiteTestScenarioRunner.sinkName, "value" -> "#result")
```
where source will produce given input data and sink expression result will be returned in results?
Now it is a "hidden knowledge"

Reply:
:+1: good point